### PR TITLE
DCT-248

### PR DESCRIPTION
**Pull request by EBEREGIT**
### `lastConsensusSecs`

Added code sample

### CODE REVIEW

File: docs/src/rsh/compute/index.md [hunk 1]
```diff
@@ -1840,6 +1840,11 @@ lastConsensusSecs()
 
 @{defn("lastConsensusSecs")} is like `{!rsh} lastConsensusTime`, except it returns the network seconds.
 
+```reach
+load: /examples/realtime/index.rsh
+range: 18 - 18
```
Comment (TheChronicMonster) [inline docs/src/rsh/compute/index.md:1845]: Same thing here. Samples should provide more context than the syntax reference. 

Let's find a different example so we're not showing a duplicate sample right after the other.

**Review by TheChronicMonster — CHANGES_REQUESTED** [review]

**Review by TheChronicMonster — APPROVED** [review]

**Comment by jeapostrophe** [comment]
Re: transition statement

**Comment by TheChronicMonster** [comment]
> 

@jeapostrophe we want to avoid a string of "In the example below" statements in the documentation. 
However, perhaps we can structure the entries as:
syntax reference
precise explanation
sample
intuitive explanation referencing sample

What do you think?

**Comment by jeapostrophe** [comment]
That's good for me.

**Comment by TheChronicMonster** [comment]
> > 
> 
> @jeapostrophe we want to avoid a string of "In the example below" statements in the documentation. However, perhaps we can structure the entries as: syntax reference precise explanation sample intuitive explanation referencing sample
> 
> What do you think?

@EBEREGIT Please update this, and the other PRs with explanations of the samples.

**Review by jeapostrophe — APPROVED** [review]

**Pull request merged**